docs/geedocs/5.2.3/answer/7160003.html

```diff
@@ -41,7 +41,7 @@ <h2>Release notes: Open GEE 5.2.3</h2>
   <h5>
     <p id="overview_5">New Features</p>
   </h5>
-    <p><strong>Example Feature</strong>. Example description</p>
+    <p><strong>C++11 support</strong>. With release 5.2.3 Open GEE C++ code now supports and defaults to compiling using the <code>-std=gnu++11</code> compile flag.  Or optionally this default can be overridden and the C++ code can be compiled using the <code>-std=gnu++98</code> flag instead.  However, 5.2.3 will be the last release to support compiling C++ code using the <code>-std=gnu++98</code> flag.  The next release will require the use of the <code>-std=gnu++11</code> flag when compiling C++ code.</p>
```

Comment:
Consider changing "Or optionally this default" to "This default" to make simpler.

Reply:
"However, 5.2.3" --> " Open GEE 5.2.3" ?

Reply:
"The next release " could be confusing. May be change to "The subsequent release after Open GEE 5.2.3"